raw-migrated-files/cloud/cloud/ec-invite-users.md

```diff
@@ -26,11 +26,13 @@ To edit a member’s roles, in the **Actions** column click the three dots and s
 
 
 ## Accept an invitation [ec-accept-invitation]
+<!-- This section has been already added by @eedugon to the cloud-account section) -->
```

Comment:
these are not showing up as comments. comments use a `%` as a prefix

<img width="963" alt="image" src="https://github.com/user-attachments/assets/a6a7853f-8865-4585-94b5-8c91b59a591c" />

Reply:
I had forgotten to update them, thanks for the catch!